device.py

```diff
@@ -44,8 +96,72 @@ def identify(self, uuid):
         }
         return self.request('/blink', 'POST', data=data)
 
-    def get_name(self, uuid):
-        return self.get(uuid)['name']
+    def rename(self, uuid, new_name):
+        if self.has(uuid):
+            params = {
+                'filter': 'uuid',
+                'eq': uuid
+            }
+            data = {
+                'name': new_name
+            }
+            return self.request('device', 'PATCH', params=params, data=data)
+        else:
+            print(Message.NO_DEVICE_FOUND.format(value=uuid,dev_att="uuid"))
 
-    def is_online(self, uuid):
-        return self.get(uuid)['is_online']
+
+    def note(self, uuid, note):
+        if self.has(uuid):
+            params = {
+                'filter': 'uuid',
+                'eq': uuid
+            }
+            data = {
+                'note': note
+            }
+            return self.request('device', 'PATCH', params=params, data=data)
+        else:
+            print(Message.NO_DEVICE_FOUND.format(value=uuid,dev_att="uuid"))
+
+    def get_display_name(self, device_type_slug):
+        device_type_found = self.config.get_device_type()
+        display_name = [device['name'] for device in device_type_found
+                        if device['slug'] == device_type_slug]
+        if display_name:
+            return display_name
```

Comment:
Seeing the list comprehension you do above (lines 128-129), ``display_name`` is a list!